backend/model/users.py

```diff
@@ -49,10 +49,16 @@ def save_to_db(self):
             db.session.add(self)
             db.session.commit()
 
-    def json(self):
-        return {"username": self.username,
-                "email": self.email,
-                "role": str(self.role)}
+    def json(self, reviews=False):
+        _ignore = self.username  # Forces execution to parse properly the class, fixing the bug of transient data
+        atr = self.__dict__.copy()
+        atr["role"] = str(atr["role"])
+        del atr["_sa_instance_state"]
+        del atr["state"]
+        del atr["date"]
+        if reviews:
+            atr['reviews'] = [review.json() for review in self.reviews]
+        return atr
```

Comment:
ara que em fixo estas retornan la contraseña, aixo es perillos. 
```suggestion
        user = {"username": self.username,	
                "email": self.email,
                "role": str(self.role)}
        if reviews:
            user['reviews'] = [review.json() for review in self.reviews    
        return user    
```

Reply:
Yep pensava que l'estava eliminant com amb el state i el date. Tens raó.